client/package.json

```diff
@@ -6,41 +6,41 @@
     "dot-prop": ">= 5.1.1"
   },
   "dependencies": {
-    "@material-ui/core": "^4.11.4",
+    "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@reach/router": "^1.3.4",
     "axios": "^0.21.1",
-    "gatsby": "^3.5.1",
-    "gatsby-image": "^3.5.0",
+    "gatsby": "^3.11.1",
+    "gatsby-image": "^3.11.0",
     "gatsby-plugin-env-variables": "^2.1.0",
-    "gatsby-plugin-typescript": "^3.5.0",
+    "gatsby-plugin-typescript": "^3.11.0",
     "react": "^17.0.2",
-    "react-cookie": "^4.0.3",
+    "react-cookie": "^4.1.1",
     "react-dom": "^17.0.2",
     "react-google-login": "^5.2.2",
     "react-slick": "^0.28.1",
     "universal-cookie": "^4.0.4"
   },
   "devDependencies": {
-    "@types/node": "^15.6.0",
-    "@types/react": "^17.0.6",
-    "@types/react-dom": "^17.0.5",
-    "@types/react-slick": "^0.23.4",
+    "@types/node": "^16.6.1",
```

Comment:
we need to revert the node version to 14

`npm install --save-dev @types/node@14`